### PR TITLE
BigInteger/BigDecimal limits backport to EE9

### PR DESCRIPTION
**Pull request by Tomas-Kraus**

### CODE REVIEW

File: impl/src/main/java/org/eclipse/parsson/api/JsonConfig.java [hunk 1]
```diff
@@ -17,9 +17,29 @@
...
+     */
+    String MAX_BIGDECIMAL_LEN = "org.eclipse.parsson.maxBigDecimalLength";
+
+    /**
+     * Configuration property to reject duplicate keys.
+     * The value of the property could be anything.
+     *
+     * @deprecated in favor of {@code jakarta.json.JsonConfig#KEY_STRATEGY}
      */
     String REJECT_DUPLICATE_KEYS = "org.eclipse.parsson.rejectDuplicateKeys";
```
Comment (lukasj) [inline impl/src/main/java/org/eclipse/parsson/api/JsonConfig.java:43]: add `@Deprecated` - the annotation

Comment (Tomas-Kraus) [reply impl/src/main/java/org/eclipse/parsson/api/JsonConfig.java:41]: It's 2.1 API so removed.

**Review by lukasj — APPROVED** [review]
LGTM

**Review by lukasj — APPROVED** [review]
LGTM

**Pull request merged**